zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -17,6 +17,8 @@ use pairing::bn256::Fr as Fp;
 use sha3::{Digest, Keccak256};
 use std::{collections::HashMap, convert::TryInto, iter};
 
+use super::table::TxLogFieldTag;
```

Comment:
merge into line 7

Reply:
updated in [77fe7a5](https://github.com/appliedzkp/zkevm-circuits/pull/335/commits/77fe7a57618c1a13a4cfee7a439daceedc2d4cdf) thanks